### ISSUE TITLE
Add a function to imshow all `fluxlike` quantities.

### ISSUE THREAD

**Issue by zkbt** [zkbt/chromatic#49]
Many datasets will have multiple arrays associate with them that are "flux-like", in the sense that there's one number for each wavelength and each time (2D dataset). The most obvious flux-like quantity is `flux` itself, but there could be many others (uncertainty, background level, column centroid, etc...). This is effectively trying to imshow every array in `rainbow.fluxlike`.

It'd be great to have a function that just makes a quick plot of all these quantities, in its own subplot. They should have a shared x-axis, so we can see things line up across all the panels. This might look something like `def imshow_fluxlike_quantities(rainbow, quantities=None):`, where `quantities = None` would mean to just plot everything and `quantities = ['flux', 'uncertainty', 'background']` would plot just a subset of quantities. 

To start developing this, the simplest approach is probably to just write a function in a notebook. Test it on some Rainbow object, and then once it behaves how you like, we can drop it into the code for the `chromatic` package. (I can post more specific instructions on how to do that below.) 

So, to get started, you might open a notebook and run  

```
from chromatic import *
rainbow = SimulatedRainbow().inject_transit()

def ...
```

and then start drafting your function.

**Comment by zkbt** [comment]
Here it will probably help to know that the existing `rainbow.imshow` function allows you to input a `quantity=` keyword, to specify any particular flux-like quantity. It might not work perfectly to handle all cases smoothly, but it's hopefully a start!

**Comment by zkbt** [comment]
Laura M is working on this!

**Comment by zkbt** [comment]
Closed by #64 !
